packages/react-reconciler/src/ReactChildFiber.js

```diff
@@ -9,8 +9,8 @@
 
 import type {ReactElement} from 'shared/ReactElementType';
 import type {ReactPortal} from 'shared/ReactTypes';
-import type {Fiber} from 'react-reconciler/src/ReactFiber';
-import type {ExpirationTime} from 'react-reconciler/src/ReactFiberExpirationTime';
+import type {Fiber} from './ReactFiber';
+import type {ExpirationTime} from './ReactFiberExpirationTime';
```

Comment:
I'm curious if this was here for a reason. @gaearon do you know of any reason why this can't be relative?

Reply:
Yes, it was because Flow has issues with symlinks on Windows.

Reply:
That said we didn't follow it consistently either.

Reply:
Okay. Reason enough to revert this? There's still the de-duplication stuff in here. We could revise this PR to remove the relative import changes.